agent/containers/images/image-test-script.sh

```diff
@@ -4,65 +4,49 @@
 # Redis server, and one or more remote
 export CONTROLLER=$(hostname -f)
 
-verified=""
-while [ -z "$verified" ]
-do
-    read -p "Specify distribution to test (or press 'ENTER' for default 'centos-8'): " DISTRO
-    if [ -z "$DISTRO" ]
-    then
-        DISTRO="centos-8"
-    fi
+DISTRO_DEFAULT="centos-8"
+TAG_DEFAULT="a933ae45e"
+REDIS_HOST_DEFAULT=$CONTROLLER
+REDIS_PORT_DEFAULT=17001
 
-    read -p "Specify image tag to test (or press 'ENTER' for default 'a933ae45e'): " TAG
-    if [ -z "$TAG" ]
-    then
-        TAG="a933ae45e"
-    fi
+while :; do
+    read -p "Specify distribution to test (or press 'ENTER' for default ${DISTRO_DEFAULT}): "
+    DISTRO=${REPLY:-${DISTRO_DEFAULT}}
+    read -p "Specify image tag to test (or press 'ENTER' for default ${TAG_DEFAULT}): "
+    TAG=${REPLY:-${TAG_DEFAULT}}
 
     printf -- "Pulling tool-data-sink image to test...\n"
-    verified=$(podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG 2> /dev/null)
-
-    if [ -z "$verified" ]
+    if podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG
     then
-        printf -- "Distro/tag combination could not be found, please try again.\n\n"
+        printf -- "Complete!\n\n" break
```

Comment:
Why are you _printing_ "break"?? Is that intentional?

Also, although the "Complete!" was there before, looking at it this way I'm not sure that's an appropriate message for validation of the first interactive parameter. We're hardly "complete" at this point. Should we instead confirm that the tag is validated and the container has been pulled?

And ... would this be a good place to use the `cmd="podman [...]" ; print ${cmd} ; ${cmd}` pattern as we do in `pbench-register-tool` at the end, so the user knows what's happening, rather than just "Pulling image"??